test_common.py

```diff
@@ -0,0 +1,219 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Dec 27 12:58:49 2016
+
+@author: anki08
+"""
+
+import numpy as np

```

Comment:
We conventionally follow the following import order:
* standard library
* other external libraries
* sklearn
* perhaps testing imports come last